vulture_os/toolkit/api_parser/cybereason/cybereason.py

```diff
@@ -136,235 +133,113 @@ def execute_query(self, method, url, query, timeout=10):
+        devicesNames = [machine['displayName'] for machine in alert['machines']]
+        alertDevicesDetails = self.getDevicesDetails(devicesNames)
+
+        alert['devices'] = []
+        for devices in alertDevicesDetails:
+            alert['devices'] += [{
+                "nat": {
+                    "ip": devices.get("internalIpAddress", '-'),
+                },
+                "hostname": devices.get("machineName", '-'),
+                "os": {
+                    "full": devices.get("osVersionType", '-')
+                },
+                "ip": devices.get("externalIpAddress", '-'),
+                "id": devices.get("sensorId", '-'),
+                "policy": devices.get("policyName", '-'),
+                "group": devices.get("groupName", '-'),
+                "site": devices.get("siteName", '-'),
+                "version": devices.get("version", '-'),
+                "organization": devices.get("organization", '-'),
+                "adOU": devices.get("organizationalUnit", '-'),
+                "domain": devices.get("fqdn", '-'),
+                "domainFqdn": devices.get("organization", '-')
+            }]
+        
+        return alert
 
-    def parseTimestamps(self, malop):
-        try:
-            timestamp_detected = float(malop['malopLastUpdateTime']) / 1000
-            malop["timestamp"] = timestamp_detected
-        except Exception as e:
-            msg = f"Error enriching timestamp detected: {e}"
-            logger.error(f"[{__parser__}]:parseTimestamps: {msg}",
-                         extra={'frontend': str(self.frontend)})
-            malop["timestamp"] = float(datetime.datetime.now().timestamp())
-        try:
-            malop["timestamp_closed"] = float(malop['closeTime']) / 1000
-        except Exception as e:
-            msg = f"Error enriching timestamp closed: {e}"
-            logger.error(f"[{__parser__}]:parseTimestamps: {msg}",
-                         extra={'frontend': str(self.frontend)})
-            malop["timestamp_closed"] = 0.0
+    def get_appliance_version(self):
+        version_url = f"{self.host}/{self.APP_VERSION_URI}"
         try:
-            malop["timestamp_start"] = float(malop['malopStartTime']) / 1000
+            ret = self.execute_query("GET", version_url, {})
+            return True, ret['data']['version']
         except Exception as e:
-            msg = f"Error enriching timestamp start: {e}"
-            logger.error(f"[{__parser__}]:parseTimestamps: {msg}",
-                         extra={'frontend': str(self.frontend)})
-            malop["timestamp_start"] = 0.0
-
-        return malop
-
-    def _descriptionsMalop(self):
-        if (len(self.DESC_MALOP_TRANSLATION) > 0):
-            return self.DESC_MALOP_TRANSLATION
-
-        url = f"{self.host}/{self.DESCRIPTION_MALOP_URI}"
-        data = self.execute_query("GET", url, None)
-        self.DESC_MALOP_TRANSLATION = data
-        return data
+            return False, str(e)
```

Comment:
Through function's usages, the version is never used anywhere (except when it contains the error string!)
function can be removed, or renamed if the ability to get the version is controlled, but its value is unused